definitions/divorce/json/AuthorisationCaseField.json

```diff
@@ -12430,5 +12430,96 @@
     "CaseFieldID": "DateCaseNoLongerEligibleForDA",
     "UserRole": "caseworker-divorce-courtadmin-la",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "26/06/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "RespSolLinkedEmail",
+    "UserRole": "caseworker-divorce-courtadmin_beta",
+    "CRUD": "CRU"
+  },
```

Comment:
@gavinclark do we want caseworkers to be able to modify this email?

Reply:
Yes. It was a request from the business. Understand they can't change case IDAM association with out devops intervention but still need to be able to amend email address when necessary.